service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -1190,7 +1201,13 @@ public ViewBuilder withLocation(String newLocation) {
     }
   }
 
-  private class BasePolarisTableOperations extends BaseMetastoreTableOperations {
+  /**
+   * An implementation of {@link TableOperations} that integrates with {@link IcebergCatalog}. Much
+   * of this code was originally copied from {@link
+   * org.apache.iceberg.BaseMetastoreTableOperations}. COPIED_COPIED_TO_POLARIS
```

Comment:
I would have expected CI to fail now that we have the `COPIED_COPIED_TO_POLARIS`, but this file in not mentioned in LICENSE :thinking:

Reply:
I added it to the license and updated the comment!

Reply:
It took me a while to see the typo :sweat_smile: